common-lib/src/main/scala/com/gu/mediaservice/model/Collection.scala

```diff
@@ -0,0 +1,18 @@
+package com.gu.mediaservice.model
+
+import org.joda.time.DateTime
+import play.api.libs.json._
+
+case class Collection(path: List[String], actionData: actionData)
+object Collection {
+  implicit def formats: Format[Collection] = Json.format[Collection]
+}
+
+// Following the crop structure
+// TODO: Use this in crop too
+case class actionData(author: String, date: DateTime)
+object actionData {
+  implicit def formats: Format[actionData] = Json.format[actionData]
+  implicit val dateWrites = Writes.jodaDateWrites("yyyy-MM-dd'T'HH:mm:ss.SSSZ")
+  implicit val dateReads = Reads.jodaDateReads("yyyy-MM-dd'T'HH:mm:ss.SSSZ")
```

Comment:
Cool, probably best as a separate PR?

Worth doing a full end-to-end as this data can show up in different places (ES index, exports S3 object metadata, etc).